docs/configuration.asciidoc

```diff
@@ -212,48 +212,19 @@ var options = {
 ----
 
 [float]
-[[error-throttling]]
-=== Error throttling
+[[event-throttling]]
+=== Event throttling
 
-To throttle the number of errors send to the APM Server, you can use the following options:
+Throttle the number of events sent to APM Server.
 
 [float]
-[[error-throttle-limit]]
-==== `errorThrottleLimit`
+[[event-limit]]
```

Comment:
We kept the config as `eventsLimit`.  Should we have to change the config?

Reply:
My mistake, I missed the `s`. Fixed now.